engine/src/main/java/org/pentaho/di/job/Job.java

```diff
@@ -1704,12 +1713,13 @@ public static String sendToSlaveServer( JobMeta jobMeta, JobExecutionConfigurati
       if ( executionConfiguration.isPassingExport() ) {
         // First export the job... slaveServer.getVariable("MASTER_HOST")
         //
-        FileObject tempFile =
-            KettleVFS.createTempFile( "jobExport", ".zip", System.getProperty( "java.io.tmpdir" ), jobMeta );
+        FileObject tempFile = KettleVFS.getInstance( DefaultBowl.getInstance() )
```

Comment:
Inn this line, instaed of DefaultBowl().getUnstance(), you can use the getBowl()

Reply:
This method is static. getBowl() is not. :(